spec/compiler/FileNameSpec.js

```diff
@@ -47,6 +47,13 @@ describe("FileName", function() {
     });
   });
 
+  describe('nextFile', function() {
```

Comment:
can you add another test for b.html, to make sure it currently returns undefined?